helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml

```diff
@@ -56,14 +56,34 @@ spec:
         service: kubelet
       record: slo_target
 
+      # -- node-exporter
+    - expr: "count(up{app="node-exporter", role!="bastion"}) by (cluster_type, cluster_id)"
+      labels:
+        class: MEDIUM
+        area: kaas
+        service: node-exporter
+      record: raw_slo_requests
+    - expr: "sum((up{app='node-exporter', role!="bastion"} * -1) + 1) by (cluster_type, cluster_id)"
```

Comment:
Can you just describe what this SLI does in a comment? Otherwise, it's fine for me to merge